src/gei/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -396,7 +398,7 @@ private void ValidateGHESOptions(MigrateRepoCommandArgs args, bool cloudCredenti
 
             if (args.UseGithubStorage)
             {
-                _log.LogWarning("Ignoring --use-github-storage flag because you are running GitHub Enterprise Server (GHES) 3.8.0 or later. The blob storage credentials configured in your GHES Management Console will be used instead.");
+                _log.LogWarning("Provding the --use-github-storage flag will supersede any credentials you have configured in your GitHub Enterprise Server (GHES) Management Console.");
```

Comment:
I think that this logic is good. To be 100% though, I would recommend testing against GHES > 3.8 by configuring the creds in the management console + providing the `--use-github-storage` to verify.